content/css/bootstrap.min.css

```diff
@@ -0,0 +1,7 @@
+@charset "UTF-8";/*!
```

Comment:
Where did these static .js files come from?  Are they just part of the ASF template?

Reply:
I've added documentation on the README about how to update these dependencies. But they do all come from the ASF infra team example